browser/android/preferences/BUILD.gn

```diff
@@ -33,8 +33,8 @@ source_set("preferences") {
 java_cpp_strings("java_pref_names_srcjar") {
   sources = [
     "//brave/components/brave_rewards/common/pref_names.cc",
-    "//brave/components/brave_vpn/pref_names.cc",
     "//brave/components/brave_shields/common/pref_names.cc",
+    "//brave/components/brave_vpn/pref_names.cc",
```

Comment:
Can we please discard this change as unrelated, thanks

Reply:
Removed linting change